sdks/python/setup.py

```diff
@@ -145,6 +145,7 @@ def get_version():
     'pymongo>=3.8.0,<4.0.0',
     'oauth2client>=2.0.1,<5',
     'protobuf>=3.12.2,<4',
+    'proto-plus>=1.7.1,<2',
```

Comment:
Note: requirements files on release branch already include this dep. Example: https://github.com/apache/beam/blob/edcfdc40954b05dd47905cbefb13d7a7437ea99b/sdks/python/container/py37/base_image_requirements.txt#L105